ntbs-service/appsettings.Development.json

```diff
@@ -7,6 +7,5 @@
     }
   },
   "ConnectionStrings": {
-    "ntbsContext": "data source=localhost;initial catalog=ntbsDev;trusted_connection=true"
-  }
+    "ntbsContext": "data source=.\\NTBSSERVER;initial catalog=ntbsDev;trusted_connection=true"  }
```

Comment:
woops! :)
Did you try out using the user-secrets bit to not have to worry about changing it back and forth?